Week2/homework/maartjes_work.js

```diff
@@ -1,47 +1,84 @@
-'use strict';
+"use strict";
 
+// 2.2
 const monday = [
   {
-    name: 'Write a summary HTML/CSS',
+    name: "Write a summary HTML/CSS",
     duration: 180
   },
   {
-    name: 'Some web development',
+    name: "Some web development",
     duration: 120
   },
   {
-    name: 'Fix homework for class10',
+    name: "Fix homework for class10",
     duration: 20
   },
   {
-    name: 'Talk to a lot of people',
+    name: "Talk to a lot of people",
     duration: 200
   }
 ];
 
 const tuesday = [
   {
-    name: 'Keep writing summary',
+    name: "Keep writing summary",
     duration: 240
   },
   {
-    name: 'Some more web development',
+    name: "Some more web development",
     duration: 180
   },
   {
-    name: 'Staring out the window',
+    name: "Staring out the window",
     duration: 10
   },
   {
-    name: 'Talk to a lot of people',
+    name: "Talk to a lot of people",
     duration: 200
   },
   {
-    name: 'Look at application assignments new students',
+    name: "Look at application assignments new students",
     duration: 40
   }
 ];
 
 const tasks = monday.concat(tuesday);
 
-// Add your code here
+let durationOfTasks = tasks.map(Element => Element.duration);
+
+ console.log(durationOfTasks);
+
+// Map the tasks to durations in hours.
+let durationInHours = durationOfTasks
+  .map(element => element / 60)
+
+ console.log(durationInHours);
+
+// Filter out everything that took less than two hours
+  
+let lessThanTwoHours = durationInHours
+    
+.filter(element => element >= 2)
+console.log(lessThanTwoHours);
```

Comment:
If you filter out all tasks that take less than two hours then you filtered array contains all elements that take two hours or more. Therefore the variable name `lessThanTwoHours` is incorrect and misleading. A name such as `durationsOTwoHoursOrMore` or something similar would be correct.

Lines 60 and 62 are part of the same statement. It is very confusing to have a blank line in between. I would append `.filter()` directly at the end of line 60.

The console.log should be present in a PR.

ESLint: the `let` should be a `const`.